tests/code_size/hello_world_wasm.js

```diff
@@ -0,0 +1,28 @@
+var c = Module;
+
+var d = new TextDecoder("utf8");
+
+function e(a) {
+    if (!a) return "";
+    for (var k = a + NaN, b = a; !(b >= k) && f[b]; ) ++b;
```

Comment:
This line keeps making me sad every day.. https://github.com/google/closure-compiler/issues/3223